src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -61,10 +61,10 @@ public void testExecuteAndPostProcess() {
                 + "instr1@course1.tmt|||New Feedback Session "
                 + "<span class=\"bold\">(ifaat tca fs)</span> for Course "
                 + "<span class=\"bold\">[idOfTypicalCourse1]</span> created.<br>"
-                + "<span class=\"bold\">From:</span> Tue Jan 31 16:00:00 UTC 2012"
-                + "<span class=\"bold\"> to</span> Wed Dec 31 16:00:00 UTC 2014<br>"
-                + "<span class=\"bold\">Session visible from:</span> Sat Dec 31 16:00:00 UTC 2011<br>"
-                + "<span class=\"bold\">Results visible from:</span> Mon Jun 22 00:00:00 UTC 1970<br>"
+                + "<span class=\"bold\">From:</span> 2012-01-31T16:00:00Z"
+                + "<span class=\"bold\"> to</span> 2014-12-31T16:00:00Z<br>"
+                + "<span class=\"bold\">Session visible from:</span> 2011-12-31T16:00:00Z<br>"
+                + "<span class=\"bold\">Results visible from:</span> 1970-06-22T00:00:00Z<br>"
```

Comment:
I remember a recent PR changed our date display format to a 'universally understood' format. Are we reversing that here?

Reply:
No; this is just the logging system. It uses `toString` on the timestamps. The default `toString` implementation for `Instant` results in this format.